docs/components/alert/elements.md

```diff
@@ -1,13 +1,13 @@
-## Syntax
+### Syntax
 
-```js
+```html
 <w-alert variant="info" show="">
   <p class="font-bold">This is "info" variant of the alert element</p>
   <p>With an additional description</p>
 </w-alert>
 ```
 
-## Accessibility
+### Accessibility
```

Comment:
IS the accessability stuff elements spesific stuff?  If not we could move it to the level outside of the tabs bar to the main document , right?

Reply:
Not something we have to do now offcourse, just thinking out loud!  :)

Reply:
I'm guessing that it at least COULD be different depending on component and implementation in each framework... even though it shouldn't. 😅 But I haven't looked into this specific (or any other accessibility section 😄), so I just left it where it was for now. 🙂 Good suggestion though. 👍